app/uk/gov/hmrc/incometaxsoftwarechoicesfrontend/views/SearchSoftwarePage.scala.html

```diff
@@ -82,15 +82,24 @@ <h2 class="govuk-heading-m">@messages("search-software.filters.heading")</h2>
       </div>
 
       <div class="govuk-grid-column-two-thirds">
-        <div id="vendor-count">
-          @softwareSearchStatusTemplate(viewModel.allInOneVendors.vendors)
-        </div>
+          <div id="vendor-count">
```

Comment:
Is it possible to move the vendor-count and software-vendor-list for the zero-results into a template file so that everything is in the one place? Happy to explain if that helps